invokeai/app/invocations/controlnet_image_processors.py

```diff
@@ -182,7 +190,11 @@ def invoke(self, context: InvocationContext) -> ControlOutput:
         return ControlOutput(
             control=ControlField(
                 image=self.image,
-                control_model=self.control_model,
+                #control_model=self.control_model,
+                control_model=ControlNetModelField(
+                    model_name="canny",
```

Comment:
@StAlKeR7779 is "canny" hard-coded here for testing purposes? what do we need to get this working with the passed-in model?

Reply:
must have been for testing, fixed